### ISSUE TITLE
A11Y: Buttons must have discernible text (Action)

### ISSUE THREAD

**Issue by OlgaLarina** [surveyjs/survey-library#9846]
https://surveyjs.io/form-library/examples/modify-titles-of-survey-elements/reactjs
https://surveyjs.io/form-library/examples/add-expandable-details-section-under-matrix-rows/reactjs

To solve this problem, you need to fix at least (1) of the following:
Element does not have inner text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
Element does not have an implicit (wrapped) <label>
Element does not have an explicit <label>
Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by JaneSjs** [comment]
TODO: add a button title and showTitle: true
